develop/dev-guide-sample-application-python.md

```diff
@@ -0,0 +1,1103 @@
+---
+title: Build a Simple CRUD App with TiDB and Golang
+summary: Learn how to build a simple CRUD application with TiDB and Golang.
+aliases: ['/tidb/dev/dev-guide-outdated-for-python-mysql-connector','/tidb/dev/dev-guide-outdated-for-sqlalchemy']
+---
+
+<!-- markdownlint-disable MD024 -->
+<!-- markdownlint-disable MD029 -->
+
+# Build a Simple CRUD App with TiDB and Python
+
+This document describes how to use TiDB and Python to build a simple CRUD application.
+
+> **Note:**
+>
+> It is recommended to use Python 3.10 or a later Python version.
+
+## Step 1. Launch your TiDB cluster
+
+<CustomContent platform="tidb">
+
+The following introduces how to start a TiDB cluster.
+
+**Use a TiDB Cloud Serverless Tier cluster**
+
+For detailed steps, see [Create a Serverless Tier cluster](/develop/dev-guide-build-cluster-in-cloud.md#step-1-create-a-serverless-tier-cluster).
+
+**Use a local cluster**
+
+For detailed steps, see [Deploy a local test cluster](/quick-start-with-tidb.md#deploy-a-local-test-cluster) or [Deploy a TiDB cluster using TiUP](/production-deployment-using-tiup.md).
```

Comment:
Do we also need to include this?
```
### 使用云原生开发环境

基于 Git 的预配置的开发环境：[现在就试试](/develop/dev-guide-playground-gitpod.md)

该环境会自动克隆代码，并通过 TiUP 部署测试集群。
```

Reply:
Nope. Actually, in the python project, the GitPod environment wouldn't be deployed by the TiUP. 'Cause the memory's limitation.